src/devtools/client/debugger/panel.d.ts

```diff
@@ -12,4 +12,5 @@ export class DebuggerPanel {
   getSourceByActorId(id?: string): any;
   getSourceByURL(url?: string): any;
   selectSource(sourceId: string, line?: number, column?: number): any;
+  getFrameId(): { asyncIndex: number; frameId?: string };
```

Comment:
We can migrate `src/devtools/client/debugger/panel.js` to .ts as well, we have large chunk of types already.

Reply:
I'll skip this one. We should be moving away from the debugger panel object anyway.